CakePHP/ruleset.xml

```diff
@@ -1,152 +1,201 @@
- </rule>
- <rule ref="CakePHP.Commenting.FunctionComment">
-  <exclude-pattern>*/tests/*</exclude-pattern>
- </rule>
- <rule ref="Squiz.NamingConventions.ValidFunctionName">
-  <exclude-pattern>*/src/*</exclude-pattern>
-  <exclude-pattern>*/tests/*</exclude-pattern>
- </rule>
- <rule ref="Generic.NamingConventions.CamelCapsFunctionName">
-  <exclude-pattern>*/src/*</exclude-pattern>
-  <exclude-pattern>*/tests/*</exclude-pattern>
- </rule>
- <rule ref="PEAR.NamingConventions.ValidFunctionName">
-  <exclude-pattern>*/src/*</exclude-pattern>
-  <exclude-pattern>*/tests/*</exclude-pattern>
- </rule>
-
- <!-- All rules in ./Sniffs are included automatically -->
+    <description>CakePHP coding standard</description>
+
+    <config name="installed_paths" value="../../slevomat/coding-standard,../../../slevomat/coding-standard,../vendor/slevomat/coding-standard" />
+
+    <exclude-pattern>\.git</exclude-pattern>
+    <exclude-pattern>/*/tmp/</exclude-pattern>
+    <exclude-pattern>tests/*/templates/*</exclude-pattern>
+
+    <rule ref="PSR2" />
+    <rule ref="PSR12" />
+
+    <!--
+    Property and method names with underscore prefix are allowed in CakePHP.
+    Not using underscore prefix is a recommendation of PSR2, not a requirement.
+    -->
+    <rule ref="PSR2.Classes.PropertyDeclaration.Underscore">
+        <severity>0</severity>
+    </rule>
+    <rule ref="PSR2.Methods.MethodDeclaration.Underscore">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Squiz.NamingConventions.ValidFunctionName.PublicUnderscore">
+        <severity>0</severity>
+    </rule>
+    <rule ref="PEAR.NamingConventions.ValidFunctionName.PublicUnderscore">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Zend.NamingConventions.ValidVariableName" />
+    <rule ref="Zend.NamingConventions.ValidVariableName.PrivateNoUnderscore">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Zend.NamingConventions.ValidVariableName.ContainsNumbers">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Zend.NamingConventions.ValidVariableName.StringVarContainsNumbers">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Zend.NamingConventions.ValidVariableName.MemberVarContainsNumbers">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Zend.NamingConventions.ValidVariableName.NotCamelCaps">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Zend.NamingConventions.ValidVariableName.MemberVarNotCamelCaps">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
+        <properties>
+            <property name="ignoreBlankLines" value="false" />
+        </properties>
+    </rule>
+
+    <!--
+    Temporarily ignore until API docblock formatting and line length issues in core code are fixed.
+    -->
+    <rule ref="CakePHP.Commenting.FunctionComment" />
+    <rule ref="CakePHP.Commenting.FunctionComment.ParamCommentNotCapital">
```

Comment:
These rules related to capitalization and full stop in docblock comments have been disabled since forever.  Adhering to these rules would surely make for more consistent api docs.

Is someone willing to update the sniffs to provide a fixer for it? If not we should just drop these sniffs.